src/lib/components/image/index.svelte

```diff
@@ -13,7 +13,7 @@
 		src: string;
 		class?: string;
 		color_palette: number[][];
-		image_loaded?: boolean;
+		image_loaded: boolean;
```

Comment:
Please dont change this

Reply:
Huh? why? its causing type issues outside component

Reply:
Resolved on https://github.com/baseplate-admin/coreproject-v4-ui/pull/77/commits/ec290b7411116a41cdcfe711b7e73b7f0f84c5be